src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -12,6 +12,7 @@
       Needs to happen in Directory.Build.targets to allow all the pkgproj's to set Version property first.
     -->
     <StableVersion Condition="'$(IncludePreReleaseLabelInPackageVersion)' != 'true'">$(Version)</StableVersion>
+    <MonoBundleLLVMOptimizer Condition="'$(MonoEnableLLVM)' == 'true'">true</MonoBundleLLVMOptimizer>
```

Comment:
Would this need another clause to test that MonoBundleLLVMOptimizer hasn't already been set? e.g. something like `'$(MonoEnableLLVM)' == 'true' and '$(MonoBundleLLVMOptimizer)' == 'false'`?

Reply:
No, properties set on the command line override those in the propertygroup, i.e. I tested, and passing `/p:MonoBundleLLVMOptimizer=false` does the expected thing here.

Reply:
Got it.